### PR TITLE
Community checklist

### PR DESCRIPTION
**Pull request by robbiecarlton**
This depends on

https://github.com/Hylozoic/hylo-node/pull/172

### CODE REVIEW

**Comment by robbiecarlton** [comment]
screen shot 2016-09-26 at 2 56 27 pm

**Comment by robbiecarlton** [comment]
@levity - good to merge?

**Comment by robbiecarlton** [comment]
I merged https://github.com/Hylozoic/hylo-redux/pull/119 into this already, so it will be easier to read once that one is merged.

**Comment by levity** [comment]
hmm conflicts here too

**Comment by robbiecarlton** [comment]
fixed

File: src/actions/index.js [hunk 1]
```diff
@@ -448,9 +450,9 @@ export function closeModal () {
   return {type: CLOSE_MODAL}
 }
 
-export function createTagInPostEditor () {
+export function createTagInModal () {
```
Comment (levity) [inline src/actions/index.js]: can this go in actions/tags?

File: src/containers/community/CommunityPosts.js [hunk 2]
```diff
@@ -33,6 +37,7 @@ class CommunityPosts extends React.Component {
 
     return <div>
       {currentUser && <PostEditor community={community}/>}
+      {canModerate(currentUser, community) && <CommunitySetup community={community}/>}
```
Comment (levity) [inline src/containers/community/CommunityPosts.js:39]: move this above the post editor

**Comment by levity** [comment]
otherwise, looks good

**Comment by levity** [comment]
👍

**Pull request merged**